db/migrate/20240606003822_reorder_flags.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class ReorderFlags < ActiveRecord::Migration[7.0]
+  def up
+    default_order = DB.query(<<~SQL)
```

Comment:
```suggestion
    current_order = DB.query(<<~SQL)
```

minor nit, since this may _not_ be the default order if admin has messed around with it